nrf_security/Kconfig

```diff
@@ -914,6 +895,36 @@ config MBEDTLS_SSL_CIPHERSUITES
 
 endif # NRF_SECURITY_ADVANCED
 
+config MBEDTLS_MPI_MAX_SIZE
+	int
+	default 1024 if MBEDTLS_RSA_C || MBEDTLS_DHM_C || CC310_MBEDTLS_DHM_C || VANILLA_MBEDTLS_DHM_C
+	default 66 if MBEDTLS_ECP_DP_SECP521R1_ENABLED
+	default 64 if MBEDTLS_ECP_DP_BP512R1_ENABLED
+	default 56 if MBEDTLS_ECP_DP_CURVE448_ENABLED
+	default 48 if MBEDTLS_ECP_DP_SECP384R1_ENABLED || MBEDTLS_ECP_DP_BP384R1_ENABLED
+	default 32 if MBEDTLS_ECP_DP_SECP256R1_ENABLED || MBEDTLS_ECP_DP_SECP256K1_ENABLED || MBEDTLS_ECP_DP_BP256R1_ENABLED || MBEDTLS_ECP_DP_CURVE25519_ENABLED
+	default 28 if MBEDTLS_ECP_DP_SECP224R1_ENABLED || MBEDTLS_ECP_DP_SECP224K1_ENABLED
+	default 24 if MBEDTLS_ECP_DP_SECP192R1_ENABLED || MBEDTLS_ECP_DP_SECP192K1_ENABLED
+	default 1024
```

Comment:
Problem with default, is that it is exactly that, a default value when Kconfig first parses the prj.conf.

This has the following two implications:
1) If application does not enable NRF_SECURITY, then everything is off, and the default value will become `1024`.
if user thereafter enables NRF_SECURITY, vanilla backend, and then selects `MBEDTLS_ECP_DP_SECP192R1_ENABLED`, then the value is still 1024
2) In an app where NRF_SECURITY is enabled by default, vanilla backend is enabled, and `MBEDTLS_ECP_DP_SECP192R1_ENABLED=y`, then default value is 24.
If user then selects `MBEDTLS_ECP_DP_CURVE448_ENABLED=y` afterwards, then ` MBEDTLS_MPI_MAX_SIZE` is still 24, but now it should be 56.

Because of 1) and 2), I believe it is safer to currently keep existing behavior, unless a better approach can be found.